docs/language-basics/scalars.md

```diff
@@ -1,28 +1,28 @@
 ---
-title: Scalars
+title: Understanding Scalars
```

Comment:
Here too this makes the side bar way harder to read

Reply:
Same with every single on under language basics, we are just going to have `Understanding` prefix everywhere which is pointless and just make navigation experience worse

Reply:
typescript use the same pattern we did https://www.typescriptlang.org/docs/handbook/interfaces.html

Reply:
<img width="301" alt="image" src="https://github.com/microsoft/typespec/assets/1031227/15cb31bc-912d-4bc9-a561-d2b89085e166">